.infra/prod/values.yaml

```diff
@@ -39,6 +39,8 @@ stack:
           value: hosted-cellxgene-$(__ARGUS_DEPLOYMENT_STAGE)
         - name: BYOD_WORKFLOWS_BUCKET_PATH
           value: byod-workflows-dev
+        - name: CELLXGENE_DISABLE_AUTH
+          value: "true"
```

Comment:
Just to double-check: setting `CELLXGENE_DISABLE_AUTH` to `true` here means the new autosave endpoints will respond 401 and the client will keep annotations read-only, so persistence stays off in production. Is the plan to flip this back once the proxy is configured to send `X-Auth-Request-User` everywhere, or did we want to stage that in a follow-up? It might be worth calling out explicitly so we don’t forget to re-enable.